include/nuttx/analog/ads1115.h

```diff
@@ -58,6 +58,8 @@
 #define ANIOC_ADS1115_READ_CHANNEL _ANIOC(AN_ADS1115_FIRST + 7)
 #define ANIOC_ADS1115_SET_HI_THRESH _ANIOC(AN_ADS1115_FIRST + 8)
 #define ANIOC_ADS1115_SET_LO_THRESH _ANIOC(AN_ADS1115_FIRST + 9)
+#define ANIOC_ADS1115_TRIGGER_CONVERSION _ANIOC(AN_ADS1115_FIRST + 10)
+#define ANIOC_ADS1115_READ_CHANNEL_NO_CONVERSION _ANIOC(AN_ADS1115_FIRST + 11)
```

Comment:
Please align all macros on this block, it will make the code more pleasant to read